test/source/tests/gmail.ts

```diff
@@ -296,6 +307,19 @@ export const defineGmailTests = (testVariant: TestVariant, testWithBrowser: Test
       await gmailPage.notPresent('.reply_message_evaluated .error_notification'); // should not show the warning about switching to encrypted reply
     }));
 
+    ava.default('mail.google.com - switch to encrypted reply for middle message', testWithBrowser('ci.tests.gmail', async (t, browser) => {
+      const gmailPage = await openGmailPage(t, browser);
+      await gotoGmailPage(gmailPage, '/FMfcgzGqRGfPBbNLWvfPvDbxnHBwkdGf'); // plain convo
```

Comment:
About a year ago we started to put test messages from the live account to the `test/source/mock/google/live-gmail-test-emails` folder in plain EML format to be able to restore the message in case it's accidentally deleted from the server. It's appropriate to have the id from the test `FMfcgzGqRGfPBbNLWvfPvDbxnHBwkdGf` in the file name.
Of course, need to take care that the message doesn't contain any sensitive data (such as email addresses of people who wouldn't want to disclose their email addresses).